src/Microsoft.AspNet.Mvc.Core/MvcOptions.cs

```diff
@@ -139,11 +139,17 @@ public int MaxModelValidationErrors
 
         /// <summary>
         /// Gets or sets the flag which causes content negotiation to ignore Accept header 
-        /// when it contains the media type */*. <see langword="false"/> by default.
+        /// when it contains the media type */*. <c>false</c>c> by default.
         /// </summary>
         public bool RespectBrowserAcceptHeader { get; set; }
 
         /// <summary>
+        /// Gets or sets the flag to exclude formatters that match only on the object type.
+        /// <c>false</c> by default.
+        /// </summary>
+        public bool ExcludeMatchOnTypeOnly { get; set; }
```

Comment:
Sure :+1: (I actually borrowed it from MVC 5, but there this setting was present on the content negotiator service and so it had some context to relate to)...some ideas:
a. `ExcludeFormatterMatchOnObjectTypeOnly`
b. A positive way of saying `FallbackOnFormatterMatchingObjectType` (with default value set to `true`)

Thoughts?